### PR TITLE
Variable ingest server output url

### PR DESCRIPTION
**Pull request by joschahenningsen**
This enables us to specify where the hls stream is located depending on the server the stream is ingested to. 

This makes test environments possible and is the first step to streaming on an independent infrastructure

Related worker pr: https://github.com/joschahenningsen/TUM-Live-Worker-v2/pull/4

### CODE REVIEW

**Comment by joschahenningsen** [comment]
just merged https://github.com/joschahenningsen/TUM-Live-Worker-v2/pull/4 This should be good to go

File: api_grpc/worker.go [hunk 1]
```diff
@@ -447,9 +449,15 @@ func notifyWorkersPremieres() {
 		}
 		workerIndex := getWorkerWithLeastWorkload(workers)
 		workers[workerIndex].Workload += 3
+		ingestServer, err := dao.GetBestIngestServer()
+		if err != nil {
+			log.WithError(err).Error("Can't find ingest server")
```
Comment (alexanderstephan) [inline api_grpc/worker.go:454]: Maybe we should stop here when an error happens?

**Review by alexanderstephan — CHANGES_REQUESTED** [review]
Looks really good, seems to match the implementation in the worker. :+1:  Just a minor comment.

File: api_grpc/worker.go (hunk 1 above)
Comment (joschahenningsen) [reply api_grpc/worker.go:454]: You're right, we absolutely should. Pushed a fix

**Review by alexanderstephan — APPROVED** [review]
Looks perfect to me now! :+1:

**Pull request merged**